tutorials/artela-evm-plus-plus.md

```diff
@@ -0,0 +1,126 @@
+# Quick Start Guide for Artela EVM++ with Rollkit
+
+<script setup>
+import constants from '../.vitepress/constants/constants.js'
+</script>
+
+This guide will assist you to quickly set up a rollup node with [Artela EVM++](https://docs.artela.network/main/Artela-Blockchain/EVM++) execution layer using Rollkit and local-DA.
+
+![evmpp](/evmpp/evmpp.png)
+
+EVM++ is a modular dual-VM execution layer that supports the dynamic creation of native extension modules for blockchain at runtime. It enables developers to leverage WasmVM to build native extensions that co-process with EVM, offering enhanced customization capabilities. More than just a dual-VM setup, these native extensions facilitate blockchain-level customization. They allow for the integration of custom logic throughout the transaction lifecycle, providing access to an expanded runtime context.
+
+## 📦 Clone the Artela Rollkit Repository
+
+Start by cloning the Artela Rollkit repository:
+
+```bash
+git clone https://github.com/artela-network/artela-rollkit.git
+cd artela-rollkit
+```
+
+## 🛥️ Run with Docker Compose
+
+Ensure Docker is installed on your system before setting up the Artela rollup node. If not already installed, download and follow the setup instructions available [here](https://www.docker.com/products/docker-desktop/).
+
+After installing Docker, run the following command to start a local development node:
+
+```bash
+docker compose up -d
+```
+
+This command launches the Artela rollup node with Rollkit and local-DA. To monitor the logs, use this command:
+
+```bash
+docker logs artroll -f
+```
+
+If you observe the following output, the local development node is running properly:
+
+```bash
+...
+7:09AM INF finalized block block_app_hash=E483920A1E1E7E492E47036300003769420813BB13BB3F25CFAFDB0DF19C144A height=3 module=BlockManager num_txs_res=0 num_val_updates=0
+7:09AM INF executed block app_hash=E483920A1E1E7E492E47036300003769420813BB13BB3F25CFAFDB0DF19C144A height=3 module=BlockManager
+7:09AM INF indexed block events height=3 module=txindex
+7:09AM INF Creating and publishing block height=4 module=BlockManager
+...
+```
+
+## 🔑 Create testing accounts
+
+Access testing accounts by entering the `artroll` Docker container:
+
+```bash
+docker exec -ti artroll /bin/bash
+```
+
+Retrieve the address and private key of testing accounts using:
+
+```bash
+# 👇 Alternatively, you can use myKey2.info, myKey3.info, myKey4.info
+./entrypoint keyinfo --file ~/.artroll/keyring-test/mykey.info --passwd test
+```
+
+This will display the testing private key and its address:
+
+```bash
+private key: { Plain Private Key in Hex Format }
+address: { Address Associated with the Private Key }
+```
+
+There are four testing accounts available in the local development node for interacting with the EVM chain.
+
+## 🧪 Build on EVM++
+
+Let’s dive into Artela EVM++ by kicking off with the Hello World project, the following code will guide you through the essentials:
```

Comment:
I can't see any connection between dev-env and tutorials.
Tutorials shows how to use public testnet, without any hints about local dev-env.
 `artroll` container doesn't expose any ports so it's not obvious how to deploy locally.

Reply:
Hey @tzdybal , thanks for the comment. The artroll container is using the host network, so it does need to expose any port explicitly (I saw your GM project is doing the same). After spinning up the artroll container, you can access port 8545 (for ETH RPC) and 8546 (For WSS) in your localhost to use the node. And for the tutorial, we will fix that.

Reply:
Just updated the links and instructions of the EVM++ tutorials. I have replaced the smart contract tutorial of ours with the `contract interaction` one under EVM section. As for the Aspect one, I've added an instruction to let the developers replace Artela Testnet node URL with `localhost:8545` to access the ETH RPC.